packages/launchpad/src/setup/ManualInstall.vue

```diff
@@ -54,16 +67,17 @@ const projectFolder = computed(() => props.gql.currentProject?.title ?? '')
 
 const props = defineProps<{
   gql: ManualInstallFragment
+  packagesInstalled: string[]
 }>()
 
-defineEmits<{
-  (event: 'back'): void
-}>()
+const toInstall = computed(() => {
+  return props.gql.wizard.packagesToInstall
```

Comment:
Do we really need this additional computed property? It's only used in one place - isn't this effectively a temp variable (that also adds another thing for the reactivity system to track?

Reply:
Yeah you are right. I will remove it.

Reply:
Yeah you are right. I will remove it.